tests/rbd/rbd_utils.py

```diff
@@ -481,6 +481,8 @@ def image_resize(self, pool_name, image_name, size, **kw):
             for passphrase in kw.get("passphrase"):
                 for key, value in passphrase.items():
                     cmd += f" --{key} {value}"
+            if kw.get("allow_shrink"):
```

Comment:
can you rebase the branch, there are some more changes in same method in master branch , Please check and do changes accordingly

Reply:
Done.